### ISSUE TITLE
Variable Rotational Relaxation

### ISSUE THREAD

**Issue by kjhigdon** [sparta/sparta#2]
## Summary

Current variable rotational relaxation approach does not reproduce the expected relaxation rate.

## Type of Issue

Physics Bug

## Detailed Description

The current version of the rotational relaxation model does not correctly reproduce the expected relaxation rate and will not reach equilibrium. In the current model, the energy of the colliding particles is used in Parker's model to calculate Z_rot, `rotphi` in the code. Using `Ec` to calculate Z_rot requires a correction factor that Boyd developed in the past. This is quickly summarized in Boyd and Schwartzentruber, _Nonequilibrium Gas Dynamics and Molecular Simulation_, pp. 254-5. 

I typically prefer using the cell temperature approach where `Tr` is replaced with the cell temperature. I suggest using the cell translational temperature instead of calculating a composite cell temperature. This is also the approach taken by Boyd and Schwartzentruber, pp. 253-4.   

The vibrational relaxation number, Z_vib or `vibphi` in the code, may also suffer from the same issue. Boyd and Schwartzentruber, pp. 260-5 discusses vibrational relaxation. In my code, I assume the translational temperature for the Millikan-White model.

## SPARTA Version

`collide_vss.cpp`

Rotation:
https://github.com/sparta/sparta/blob/45e17cdbda3845e40ed5aaf44399323c365b2941/src/collide_vss.cpp#L773

Vibration:
https://github.com/sparta/sparta/blob/45e17cdbda3845e40ed5aaf44399323c365b2941/src/collide_vss.cpp#L785

## Expected Behavior

Thermal equilibrium should be obtained in a relaxation case.

## Actual Behavior

When initializing a simple heat bath simulation at equilibrium, the temperatures diverge to some steady nonequilibrium state when using variable rotational relaxation. The correct result is observed when using a constant relaxation number.

## Steps to Reproduce

Run a heat bath simulation with N2 where all temperatures start at an equilibrium of ~10,000 K.

**Comment by stanmoore1** [comment]
@kjhigdon do you have an input script for the N2 heat bath? That would be really helpful.

**Comment by kjhigdon** [comment]
This equilibrium case is set up for initializing N2 at 13,000 K. I included the output from when I ran it. 

Again, I didn't clean it up too much so there might be some unnecessary steps in the input file. It still has the capability to be easily switched to a N2 - N relaxation/chemistry case which are also good benchmark tests. If you want to compare with some published relaxation benchmarks. The test conditions I've set up are very close to what's published in Boyd and Schwartzentruber's book.

[Equil_test.zip](https://github.com/sparta/sparta/files/2294012/Equil_test.zip)
